backend/src/main/java/gov/cdc/usds/simplereport/db/model/BaseTestInfo.java

```diff
@@ -19,19 +19,19 @@
   @JoinColumn(name = "patient_id", updatable = false)
   private Person patient;
 
-  @ManyToOne(optional = false)
+  @ManyToOne(optional = false, fetch = FetchType.LAZY)
```

Comment:
This is reasonable (we don't look at this at all in the common case) but also fairly low-benefit, since in the common case there is only one value to fetch.